src/Printer/CollectorMetadataPrinter.php

```diff
@@ -56,6 +54,7 @@ public function printArgTypesAsString(MethodCall $methodCall, ExtendedMethodRefl
                 return ResolvedTypes::UNKNOWN_TYPES;
             }
 
+            // @phpstan-ignore phpstanApi.instanceofType
```

Comment:
In this case there is no good alternative to checking that it is an instance of IntersectionType (at least I was not able to find one), so I have added a ignore comment

Reply:
Better place it in `phpstan.neon`, so we have all ignores in one place.